packages/store/src/operators/of-action.ts

```diff
@@ -74,6 +74,26 @@ export function ofActionErrored(...allowedTypes: any[]) {
   return ofActionOperator(allowedTypes, [ActionStatus.Errored]);
 }
 
+/**
+ * RxJS operator for selecting out specific actions.
+ *
+ * This will grab an action and return true when dispatched and false when completed
+ */
+export function ofActionExecuting(allowedType: any) {
+  return (source: Observable<ActionContext>) => {
```

Comment:
`return merge` is less readable. Lambda function explicitly returns value after `=>` + less nesting inside of the `merge`.

```suggestion
  return (source: Observable<ActionContext>) => merge(
```